netbox/netbox/navigation/menu.py

```diff
@@ -58,6 +58,7 @@
             label='Devices',
             items=(
                 get_model_item('dcim', 'device', 'Devices'),
+                get_model_item('dcim', 'virtualdevicecontext', 'Virtual Device Contexts'),
```

Comment:
IMO this should go somewhere else in the menu. Seems random to have a virtual resource listed between two physical ones.

Reply:
Moved it just below virtual chassis